ui/console/pom.xml

```diff
@@ -79,4 +104,39 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>dev</id>
```

Comment:
So only when I specify -Pdev, the stuff is not deleted? Would it make sense the other way around?